tool/grind.dart

```diff
@@ -141,16 +150,24 @@ void all() {}
 
 @Task('Run the Dart formatter.')
 void format() {
-  run('dart', arguments: ['format', '--fix', '.']);
+  run('dart', arguments: ['format', '.']);
+  run('dart', arguments: ['fix', '--apply', '.']);
```

Comment:
I think it's better to apply the fix first and then format?

```suggestion
  run('dart', arguments: ['fix', '--apply', '.']);
  run('dart', arguments: ['format', '.']);
```

Reply:
There are arguments both ways. Fixing first will properly handle lines that are made longer by fixed, while formatting first will correctly handle things like single-line `if` statements that are pushed onto multiple lines by the formatter. Maybe we should just do fix-format-fix, since the second fix shouldn't ever cause a need for a second format, although that's going to be wasted work most of the time.